test/interop/python/multilocale/boolFunctions.chpl

```diff
@@ -0,0 +1,15 @@
+/* Tests accepting a bool */
+export proc takesBool(x: bool) {
+  writeln(x);
+}
+
+/* Tests returning a bool */
+export proc getBool(): bool {
```

Comment:
What C type does `bool` map to?

Reply:
`chpl_bool`, and that type accepts `bool` from `stdbool.h`